POLICY.md

```diff
@@ -0,0 +1,65 @@
+    *	Support for new board. If the new board is not widely used, doesn’t have a manufacturer webpage, etc, then it isn’t desirable to support it. If the new board is essentially a duplicate of another, it isn’t desirable to duplicate the existing one.
+9.	Feature requests or changes that are meant to address a very specific/limited use case, especially if at the expense of increased code complexity, may be denied, or may be required to be redesigned, generalized, or simplified
+10.	Feature requests that are not accompanied by a PR:
+    *	could be closed immediately (denied)
+    *	could be closed after some predetermined period of time (left as candidate for somebody to pick up)
+    *	could be deemed interesting enough to work on, but without a specific project or target, and hence accumulated in a long-term feature request list. Such feature requests should in general not be targeted for a deadline or release.
+11.	In some cases, feedback may be requested from the requestor, either as additional info for clarification, additional testing, or other. If no feedback is provided after 30 days, the issue may be closed by a contributor.
+
+# Compatibility
+1.	Compatibility with the Arduino build system is first priority. Compatibility with PlatformIO and make are also maintained, but are second priority.
+2.	Feature requests should consider compatibility with Arduino
+    *	ESP-specific APIs should be added with care, and should be marked as such (Example: ESP8266WiFi)
+    *	APIs of common libraries should maintain compatibility (Example: Wire, SPI, Servo)
+    *	ESP-specific extensions to compatible APIs are ok, especially if required to fully use certain peripherals, but such functions should be clearly marked as ESP-specific
+3.	When making changes that are likely to impact PlatformIO or make, relevant people should be notified. Check whether some corresponding changes are needed on the build system side. When an issue related to one of these build systems is reported, redirect the issue reporter to the respective issue tracker.
+4.	The core libs are implemented as a wrapper layer over the Espressif SDK. Due to the requirements and limitations imposed by the SDK, there are inherent differences between the behavior of this core and the standard Arduino core (Example: using long delay()s is not allowed here). Compatibility can’t be maintained in such cases, and differences should be clearly documented.
+
+# Pull requests
+1.	All pull requests should undergo peer review by at least one contributor other than the requestor
+2.	All pull requests should consider updates to the documentation
+3.	All pull requests should consider updates to regressions, where possible
+4.	Pull requests that address an outstanding issue, particularly an issue deemed to be severe, should be given priority
+5.	If a feature requests is accepted, the PR should then be updated based on feedback, then merged.
+
+# Other
```

Comment:
Suggest moving each item from this section into a new issue, or into a "project". Unlike all previous sections, this is more of a "here is where we want to go, but something needs to be done about this", rather than "here are the rules".

Reply:
@igrr I don't quite understand what you mean. Could you please elaborate?

Reply:
Do you mean eliminate the section altogether, and create projects for the three topics?